sippy-ng/src/component_readiness/CompReadyMainInputs.js

```diff
@@ -80,8 +83,44 @@ export default function CompReadyMainInputs(props) {
       ></AdvancedOptions>
     </div>
   )
+
+  // eslint-disable-next-line react/prop-types
+  function ViewPicker({ enabled }) {
```

Comment:
This should probably be in it's own file, IMHO it makes it a little easier to navigate the code base without an IDE telling you where this is

Reply:
Done